flows/atd-traffic-incident-reports/traffic-incident-reports.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+"""
+Name: Traffic Incident Reports to postgrest
+Description: wrapper etl for atd-traffic-incident-reports docker image \
+connects to oracle db and updates postrgrest with incidents
+Schedule: "*/5 * * * *"
+
+Create Deployment:
+$ prefect deployment build flows/atd-traffic-incident-reports/traffic-incident-reports.py:traffic_incidents_flow \
+--name "Austin Travis County Traffic Incident Reports" --pool atd-data-03 \
+--cron "*/5 * * * *" -q default -sb github/atd-prefect-main-branch \
+--description "Wrapper ETL for the https://github.com/cityofaustin/atd-traffic-incident-reports docker image connects to oracle db and updates postrgrest with incidents" \
+-o "deployments/traffic-incident-reports.yaml" --skip-upload
+ 
+$ prefect deployment apply deployments/traffic-incident-reports.yaml
+
+"""
+
+import prefect
+import docker
+from datetime import datetime, timedelta
+
+# Prefect
+from prefect import flow, task, get_run_logger
+from prefect.blocks.system import JSON
+
+# Docker settings
+docker_env = "production"
+docker_image = f"atddocker/atd-traffic-incident-reports:{docker_env}"
+
+
+@task(
+    name="get_env_vars",
+    retries=10,
+    retry_delay_seconds=timedelta(seconds=15).seconds,
+)
+def get_env_vars():
+    # Environment Variables stored in JSON block in Prefect
+    return JSON.load("atd-traffic-incident-reports").dict()["value"]
+
+
+@task(
+    name="pull_docker_image",
+    retries=1,
+    retry_delay_seconds=timedelta(minutes=5).seconds,
+)
+def pull_docker_image():
+    client = docker.from_env()
+    client.images.pull(docker_image)
+    return True
+
+
+# Retrieve the provider's data
+@task(
+    name="update_postgrest_with_traffic_incidents",
+    retries=2,
+    retry_delay_seconds=300,
```

Comment:
Since the schedule is every 5 minutes I think you should reduce this to like 30 seconds so it doesn't step on the toes of the next flow run.

Reply:
Good call -- I also dropped the retry to just once